main/zoe/api/contract-support/price-authority.md

```diff
@@ -136,3 +136,73 @@ const quote = E(pAuthority).quoteWhenLTE(
     amountMath.make(brands.Out, 974n),
   );
 ```
+## `MutableQuote` object
+
+Use a `MutableQuote` when you expect to make multiple calls, replacing the trigger
+value. If you just need a single quote, and won't change the trigger level, then use
+a non-mutable quote.
+
+There are four *mutable quote* methods, which return a `MutableQuote` object with the methods:
+- `cancel(e)`
+  - `e``{ String }`
+  - Causes the `Promise` to reject with the message `e`. When the promise is used with a `.then()`
```

Comment:
@erights said yesterday that we should pervasively replace .then with `E.when()`. Might as well start here.